bin/fuel-core/chainspec/testnet/state_config.json

```diff
@@ -13,6 +13,9 @@
       "balances": []
     }
   ],
+  "blocks": [],
+  "block_consensus": [],
+  "transactions": [],
```

Comment:
I think we don't need to migrate blocks, transactions, and consensus information via `StateConfig`. The `StateConfig` is only used for the JSON serialization and only used to generate a genesis block in a user-friendly form. For the real re-genesis process, we use only parquet.

Reply:
It doesn't hurt, but I prefer to avoid dead code since we need to support it, plus it affects our interfaces=)

I don't see any usage of it right now. Transactions are unique and can't be used by other transactions(only `UtxoId` contains `TxId`, but we can use the `coins` field to generate any input coins). Blocks and consensus info contain only transactions and are not used by transactions as well, so it is also hard to imagine why we need it for `StateConfig`.

Reply:
3a721e5bf